google/ggrc-core
# QUICK-FIX Use then for makeGapiAuthRequest

**Important:** after finishing kokoro checks the target branch should be changed to `canjs-3-migration`

# Issue description

Error appears on attempt to assign folder.

# Steps to test the changes

1. Open a business object (for instance standard)
2. Click on button "Assign folder"

Actual result: script error appears.
Expected result: authorize or `Select folder` modal appears.

# Solution description

`gapi.auth.authorize` returns custom promise object which does not contain `pipe` method. Solution is to replace `pipe` to `then` in `runAuthorization`.

# Sanity checklist




# PR Review checklist
